hub/management/commands/import_wildlife_trust_regions.py

```diff
@@ -0,0 +1,43 @@
+from django.conf import settings
```

Comment:
did you mean to include a whole file in here?

Reply:
I did not